modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java

```diff
@@ -156,10 +156,6 @@ public ObjectField postObjectDefinitionObjectField(
 			throw new UnsupportedOperationException();
 		}
 
-		ObjectFieldUtil.addListTypeDefinition(
-			contextUser.getCompanyId(), _listTypeDefinitionLocalService,
-			_listTypeEntryLocalService, objectField, contextUser.getUserId());
-
```

Comment:
@murilostodolni Why aren't we setting the `ListTypeDefinitionId` to `objectField` as in `putObjectField` ([here](https://github.com/liferay-objects/liferay-portal/blob/f947135567b7376d6e5b58f750016d876d1569e4/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java#L211-L219))?

Reply:
@carolmariaabb because in the postObjectDefinitionObjectField don't exist field. In this case we don't need validate, only create ou get listTypeDefinitionId [here](https://github.com/liferay-objects/liferay-portal/pull/1329/files/f947135567b7376d6e5b58f750016d876d1569e4#:~:text=public%20static%20long%20addListTypeDefinition). It's make sense?

Reply:
It makes sense! Thanks @murilostodolni